lib/mastodon/media_cli.rb

```diff
@@ -113,5 +113,24 @@ def usage
       say("Imports:\t#{number_to_human_size(Import.sum(:data_file_size))}")
       say("Settings:\t#{number_to_human_size(SiteUpload.sum(:file_file_size))}")
     end
+
+    desc 'lookup', 'Lookup a toot by passing a media URL'
+    def lookup
+      prompt = TTY::Prompt.new
+
+      exit(1) unless (url = prompt.ask('Please enter a URL to the media to lookup:', required: true))
```

Comment:
what's the point of the exit(1) here? you'll never get a nonempty string back from `required: true`

Reply:
You're right that's unnecessary, modifying my code to account for that.